packages/gas-fee-controller/src/gas-util.test.ts

```diff
@@ -223,4 +225,114 @@ describe('gas utils', () => {
       expect(normalizeGWEIDecimalNumbers(NaN)).toBe('0');
     });
   });
+
+  describe('calculateTimeEstimate', () => {
+    const mockGasEstimates = mockEIP1559ApiResponses[0] as GasFeeEstimates;
```

Comment:
Hmm, this does not seem ideal. Perhaps the fixture data could be expanded to include the required types? It can be copied from the endpoint directly: https://gas-api.metaswap.codefi.network/networks/1/suggestedGasFees

Reply:
Good idea, much better than a forced type cast